commands/database-push.js

```diff
@@ -62,13 +61,11 @@ module.exports = new Command("database:push <path> [infile]")
             }
 
             var consoleUrl = utils.consoleUrl(options.project, "/database/data" + path + body.name);
-            var refurl =
-              utils.addSubdomain(api.realtimeOrigin, options.instance) + path + body.name;
 
             utils.logSuccess("Data pushed successfully");
             logger.info();
             logger.info(clc.bold("View data at:"), consoleUrl);
-            return resolve(new Firebase(refurl));
+            return resolve();
```

Comment:
```suggestion
            return resolve({key: body.name});
```

Reply:
Sounds reasonable to return the key. It's only a string and at least that way we are not initializing the firebase client.
Thanks